globus_automate_client/queues_client.py

```diff
@@ -130,17 +133,17 @@ def receive_messages(
         params: Dict[str, Any] = {"max_messages": max_messages}
         if receive_request_attempt_id is not None:
             params["receive_request_attempt_id"] = receive_request_attempt_id
-        return self.get(f"/queues/{queue_id}/messages", params=params)
+        return self.get(f"/queues/{queue_id}/messages", query_params=params)
 
     def delete_messages(
         self, queue_id: str, receipt_handles: List[str]
     ) -> GlobusHTTPResponse:
         self.authorizer = get_authorizer_for_scope(QUEUES_RECEIVE_SCOPE)
         body = {"data": [{"receipt_handle": rh} for rh in receipt_handles]}
```

Comment:
I guess the convention of calling this `data` elsewhere had kind of been put in place...

Reply:
It seems to vary between methods; I saw `req_body`, `data`, `body`, and `payload` in other places but didn't standardize this throughout.